build/e2e_test.go

```diff
@@ -457,20 +457,9 @@ func TestPartialSuccess(t *testing.T) {
 
 	for i := 0; i < 4; i++ {
 		nm := fmt.Sprintf("F%d", i)
-
-		// no error checking: the 2nd call will fail
 		_ = b.AddFile(nm, []byte(strings.Repeat("01234567\n", 128)))
-		if i == 1 {
-			// force writes to fail.
-			if err := os.Chmod(dir, 0o555); err != nil {
-				t.Fatalf("chmod(%s): %s", dir, err)
-			}
-		}
-	}
-
-	if err := os.Chmod(dir, 0o755); err != nil {
-		t.Fatalf("chmod(%s, writable): %s", dir, err)
 	}
+	b.buildError = fmt.Errorf("any error")
```

Comment:
accidental inclusion?

Reply:
no, that's on purpose. I left a comment in the PR description

> @ggilmore I had to update a test which relied on manipulating folder permissions. The test fails inside a docker container because of the way docker handles permission bits on folders. For example this works inside a container